atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -655,6 +655,13 @@ public void testWriteWriteConflictsDeletedThrow() {
         }
     }
 
+    @Test (expected = IllegalArgumentException.class)
+    public void disallowPutOnEmptyObject() {
+        final Cell cell = Cell.create("row1".getBytes(), "column1".getBytes());
+        Transaction t1 = txManager.createNewTransaction();
+        t1.put(TABLE, ImmutableMap.of(cell, new byte[0]));
```

Comment:
Can we use EMPTY_BYTE_ARRAY like the deleted lines above? I like it having a name.